CondTools/L1Trigger/src/DataWriter.cc

```diff
@@ -84,7 +84,7 @@ namespace l1t {
 
     if (poolDb->isNewTagRequest(esRecordName)) {
       sinceRun = poolDb->beginOfTime();
-      poolDb->createNewIOV(payloadToken, sinceRun, esRecordName);
+      poolDb->createOneIOV(payloadToken, sinceRun, esRecordName);
```

Comment:
This replacement is wrong. "CreateOneIOV" takes a payload object as first parameter, while in the replaced methods the method called ( admittedly confusing, but this is exactly the reason we are doing this work...) refers NOT to the signature "createNewIOV(const T* payloadPtr, cond::Time_t firstSinceTime, cond::Time_t, const std::string& recordName)", the only one taking a payload object, but to the signature "createNewIOV(const std::string& firstPayloadId, cond::Time_t firstSinceTime, const std::string& recordName)".
Therefore, we don't need to replace it. I'll change that name in a further PR later.